### PR TITLE
removed layernorm

### LINKED ISSUE #172: Graduate LayerNormalization layer to TF core.
**System information**
- TensorFlow version (you are using):
- TensorFlow Addons version:
- Is it in the tf.contrib (if so, where):
- Are you willing to contribute it (yes/no):
- Are you willing to maintain it going forward? (yes/no):

**Describe the feature and the current behavior/state.**
LayerBatchNormalization has been moved to core here: https://github.com/tensorflow/tensorflow/blob/43a255d44035f2a68585288876e2ccbb77cb70de/tensorflow/python/keras/layers/normalization.py#L807

Can we graduate it from Addons?

**Will this change the current api? How?**
Yes, this will delete the LayerBatchNormalization API in Addons.

**Who will benefit with this feature?**

**Any Other info.**

### PR DESCRIPTION
**Pull request by Smokrow**
I removed LayerNorm from layers/Normalizations.py since it moved to core. This resolves #172

### CODE REVIEW

**Comment by Smokrow** [comment]
damn I knew I forgot something :)
